mantidproject/mantid
# D2B load single point scans

**Description of work.**
Fixes `LoadILLDiffraction` to load also scans with single time index.

**Report to:** [ILL/G. Cuello]

**To test:**
```
ws = Load(Filename='ILL/D2B/543614.nxs')
```
Should load with no error, the data file is in unit tests data.



Fixes #25981 . 




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
